src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -92,6 +92,7 @@ public static class BuiltinNodeCategories
         public const string REVIT_PARAMETERS = "Revit.Parameters";
         public const string REVIT_BAKE = "Revit.Bake";
         public const string REVIT_API = "Revit.API";
+        public const string REVIT_ELEMENTS_DIVIDEDPATH_ACTION = "Revit.Elements.DividedPath.Actions";
```

Comment:
@Benglin , I've changed my [Revit PR](https://github.com/DynamoDS/DynamoRevit/pull/194). But in my opinion it's better to use string from BuiltinNodeCategories(as it was before), than use this hardcoded string. But I couldn't find any other place to store it.